google/cloud/bigtable/tests/data_integration_test.cc

```diff
@@ -122,6 +124,28 @@ TEST_P(DataIntegrationTest, TableBulkApply) {
   CheckEqualUnordered(expected, actual);
 }
 
+TEST_P(DataIntegrationTest, TableBulkApplyThrottling) {
+  if (GetParam() == "with-data-client") GTEST_SKIP();
+
+  // Make a custom table with throttling enabled.
+  auto table = Table(
+      MakeDataConnection(Options{}.set<experimental::BulkApplyThrottlingOption>(
+          absl::monostate{})),
```

Comment:
Q.  Would this test fail without the option?

Reply:
No, but maybe something would break if we implemented the feature incorrectly?

The behavior of the feature is observable. We could time the test to make sure it takes at least the initial period * N requests. I am meh on adding that sort of check though.

The clock we would use in our test would be the same clock as is used in the limiter. So I do not think we would see time based errors/flakes.

Anecdotally from my own testing, the server takes a few requests before it sends any rate limiting information back. I don't want to rely on that behavior though. If the server tells us to speed up immediately, then the test may take < initial period * N requests, breaking the expectation.

Reply:
> maybe something would break if we implemented the feature incorrectly?

Maybe?  I guess code coverage is a good thing, but we should probably try not to leave an impression that we're testing any behavior.

Reply:
I added a comment to leave no impression that we're testing any behavior.